task_hive/.github/workflows/sonarcloud.yml

```
@@ -1,40 +1,48 @@
     runs-on: ubuntu-latest
-
     steps:
-      - name: Checkout repository
-        uses: actions/checkout@v3
-
-      - name: Setup Java
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)